autoload/neomake.vim

```diff
@@ -531,6 +536,7 @@ function! s:Make(options) abort
         let s:make_id += 1
         let s:make_options[s:make_id] = {
                     \ 'cwd': getcwd(),
+                    \ 'verbosity': get(g:, 'neomake_verbose', 1) + &verbose,
```

Comment:
> So, doing the addition isn't a terrible idea to me.

I think so, too.. I'll sleep over it.
Thanks for your valuable feedback once again! :)